hail/src/test/scala/is/hail/expr/ir/MatrixIRSuite.scala

```diff
@@ -273,4 +273,35 @@ class MatrixIRSuite extends SparkSuite {
       assert(values.isSorted && values.length == 11, n -> shuffle)
     }
   }
+
+  @Test def testMatrixWrite() {
+    // Native Writer
+    val range = MatrixTable.range(hc, 11, 3, Some(10))
+    var path = tmpDir.createLocalTempFile(extension = "mt")
+    Interpret(MatrixWrite(range.ast, MatrixNativeWriter(path)))
+    val read = MatrixTable.read(hc, path)
+    assert(read.same(range))
+
+    // Export VCF
+    val vcf = hc.importVCF("src/test/resources/sample.vcf")
+    path = tmpDir.createLocalTempFile(extension = "vcf")
+    Interpret(MatrixWrite(vcf.ast, MatrixVCFWriter(path)))
+
+    // Export PLINK
+    val plinkPath = "src/test/resources/skip_invalid_loci"
+    val plink = hc.importPlink(plinkPath + ".bed", plinkPath + ".bim", plinkPath + ".fam", skipInvalidLoci = true)
+    val plinkIR = MatrixMapRows(plink.ast, InsertFields(Ref("va", plink.rvRowType), // FIXME: Make importPlink output varid as field name instead of rsid
```

Comment:
I think this FIXME is gonna get lost, let's create an issue and assign it to someone.

Reply:
this is a breaking change, we can't do it until 0.3.